### PR TITLE
Added tooltip sense & add missing tooltips

### PR DESCRIPTION
**Pull request by parasharrajat**
### Details

- [x] Instant Tooltip.
- [x] Add missing tooltips.
- [x] cleanup.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5347

### Tests | QA Steps
1. Open NewDot on Web.
2. hover over any target that has a tooltip.
3. Now move your cursor to another tooltip target within 1 sec.
4. Tooltip should be shown immediately.
5. Change multiple targets and the same behavior should be shown.
6. Now move your cursor from tooltip target to an area that does not have any tooltip attached.
7. Wait for 1 sec, move back your cursor over the tooltip target.
8. Tooltip should show with animation and delay.
9. Perform repeat steps 3-8 to observe the changed behavior (Instant tooltips).

### Tested On

- [x] Web
- [ ] Mobile Web (does not have tooltips)
- [x] Desktop
- [ ] iOS (does not have tooltips)
- [ ] Android (does not have tooltips)

#### Web

https://user-images.githubusercontent.com/24370807/136609468-3c5fb333-573f-45bd-a4a4-26660eea154f.mp4

### CODE REVIEW

File: src/components/Tooltip/TooltipSense.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    debouncedDeactivate.cancel();
+}
+
+function isActive() {
+    return active === true;
+}
+
+function deactivate() {
+    return debouncedDeactivate();
+}
```
Comment (mountiny) [inline src/components/Tooltip/TooltipSense.js:20]: NAB: A bit nitty-gritty, but I would keep the order alphabetical same as you have it in the export default on lines 23-25.

**Review by mountiny — COMMENTED** [review]
@parasharrajat Tested this and love it! Great job Rajat 🎉 

I have left just one styling comment to swap the order of the functions, otherwise code LGTM and feel free to continue with the other steps. Thank you for working on this improvement!

**Comment by mountiny** [comment]
Thank you, I mean, it is not n6 polish so no need to rush on this one especially if you have some other issues, which are considered to be n6 polish. But feel free to go ahead for sure :)

**Review by mountiny — APPROVED** [review]
Code looks good to me, great job Rajat. As mentioned in the parent issue, I will keep this on hold for n6 as this was not requested as part on the polish and we want to avoid potential regressions as much as possible before the release.

Thank you very much for your work on this PR.

**Comment by parasharrajat** [comment]
@mountiny All set.

**Review by mountiny — APPROVED** [review]
Thank you very much for your work on this and your patience @parasharrajat 🙇 Great job!

**Pull request merged**